docs/Using-Fleet/REST-API.md

```diff
@@ -3227,8 +3228,8 @@ Returns a list of the hosts that belong to the specified label.
 | query                    | string  | query | Search query keywords. Searchable fields include `hostname`, `machine_serial`, `uuid`, and `ipv4`.                                                                                                                         |
 | team_id                  | integer | query | _Available in Fleet Premium_ Filters the hosts to only include hosts in the specified team.                                                                                                                                |
 | disable_failing_policies | boolean | query | If "true", hosts will return failing policies as 0 regardless of whether there are any that failed for the host. This is meant to be used when increased performance is needed in exchange for the extra information.      |
+| mdm_id                  | integer | query | The ID of the _mobile device management_ (MDM) solution to filter hosts by (that is, filter hosts that use a specific MDM provider and URL).      |
```

Comment:
This reminds me, will you ever need to call `GET /api/v1/fleet/hosts/count` with a combination of `label_id` and `mdm_id`? There are 3 API endpoints that are closely related but slightly different, List Hosts, List Hosts in a Label, and Count Hosts (which is the Count equivalent of List Hosts/List Hosts in a Label, but combines label and label-less in a single endpoint). 

If you ever need to have a count for label+mdm id, then you'll need a change in that endpoint too.

Reply:
Agreed, worth applying to the `GET fleet/hosts/count` endpoint so that we have that capability.

Reply:
Never mind, I see the `GET /fleet/hosts/count` endpoint already supports combining `mdm_id` with `label_id`.